### PR TITLE
FIX: Insets for FloatButton for RNav 6

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/miliomeucgxft7lmwo4lstogxq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11867)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @limpbrains] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/limpbrains

File: components/FloatButtons.tsx [hunk 1]
```diff
@@ -14,12 +15,10 @@ const cStyles = StyleSheet.create({
   },
   rootAbsolute: {
     position: 'absolute',
-    bottom: 30,
   },
   rootInline: {},
   rootPre: {
     position: 'absolute',
-    bottom: -1000,
```
Comment (limpbrains) [inline components/FloatButtons.tsx:22]: this is used to render component outside of visible scope, to determine it's size and prevent blinking.
Are you sure this is no longer needed?

**Review by limpbrains — CHANGES_REQUESTED** [review]

File: components/FloatButtons.tsx (hunk 1 above)
Comment (marcosrdz) [reply components/FloatButtons.tsx]: Reverted

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/2divh6os5jn6nrecotflgqmoq4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11869)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/qdvdudzbgxz75w7rhickfsdj5y?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/11868)

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**